resolwe_bio/descriptors/reads_descriptorschema.yaml

```diff
@@ -0,0 +1,20 @@
+- model: flow.descriptorschema
```

Comment:
There is no need for both, the reads_descriptorschema.yaml and reads_template.yml.